modules/src/ics02_client/error.rs

```diff
@@ -48,6 +48,12 @@ pub enum Kind {
 
     #[error("mismatch between client and arguments types, expected: {0:?}")]
     ClientArgsTypeMismatch(ClientType),
+
+    #[error("unknown client message type: {0}")]
```

Comment:
Hmm seems like `UnknownClientMessageType` and `InvalidSigner ` are not used anywhere.